README.md

```diff
@@ -1,27 +1,35 @@
-# ZingerUi
+<h1 align="center">
+  Zinger Admin Panel
+</h1>
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 9.1.0.
+This repo contains admin panel for both shop admin and zinger admin to manage zinger core. It provides functionality to manage shops, inventory, users etc.
```

Comment:
A web dashboard for shop owners and platform admin to manage the Zinger core operations. It also provides a functionality to manage shops, inventory, users, etc.